components/raftstore-v2/src/batch/store.rs

```diff
@@ -143,6 +171,10 @@ impl<EK: KvEngine, ER: RaftEngine, T: Transport + 'static> PollHandler<PeerFsm<E
         };
         let mut delegate = PeerFsmDelegate::new(peer, &mut self.poll_ctx);
         delegate.handle_msgs(&mut self.peer_msg_buf);
+        delegate
+            .fsm
+            .peer_mut()
+            .handle_raft_ready(delegate.store_ctx);
```

Comment:
Why not do it inside `handle_msgs`?

Reply:
Because handle raft ready is not handling messages.

Reply:
`poller::handle_normal` is defined as handling normal messages, Handling ready should be counted as part of handling a message. I think it would be better to avoid leaking details like "ready" into a message handling module `batch`.

Reply:
No, handling ready is a separate operation. It can be executed after handling a batch of messages, it can also be executed after handling several batch of messages. For example, in old versions, it's called in `end` instead of `handle_normal`. The timing of processing ready is subject to change for performance evaluation.